src/components/Customer/EditOktaInfoForm/EditOktaInfoForm.jsx

```diff
@@ -46,14 +40,14 @@ const EditOktaInfoForm = ({ initialValues, onSubmit, onCancel }) => {
               <h2>Your Okta Profile</h2>
               <p>
                 This is the information stored in your Okta Profile used for logging into MilMove. If you wish to change
-                any information, you can do so here by changing these fields and clicking <b>Save</b>.
+                any information, you can do so here by changing the appropriate fields and clicking <b>Save</b>.
               </p>
               <p>
                 If you need to update your security methods or set up additional security methods, you can update/add
                 those in the <Link to="https://test-milmove.okta.mil/enduser/settings">Okta Dashboard.</Link>
```

Comment:
We'll want some way of using a parameter that changes from test/prod. We have the tenant URL in chamber, this can be passed in with /enduser/settings appended. This would be good for testers/users experimenting with the staging and prod environments. Noting no backend functionality has been added yet though

Reply:
Wrong line used, see `those in the <Link to="https://test-milmove.okta.mil/enduser/settings">Okta Dashboard.</Link>`